### PR TITLE
modified owners

### PR DESCRIPTION
**Pull request by capri-xiyue**
## Proposed Changes

- Modified owners
-
-

**Release Note**

```release-note

```

### CODE REVIEW

**Comment by capri-xiyue** [comment]
assign @nachocano

File: OWNERS [hunk 1]
```diff
@@ -1,15 +1,17 @@
 approvers:
-- evankanderson
 - grantr
 - Harwayne
 - nachocano
+- capri-xiyue
+- ian-mi
```
Comment (nachocano) [inline OWNERS]: let's add @yolocs as well

Comment (nachocano) [reply OWNERS]: both in approvers and reviewers

Comment (capri-xiyue) [reply OWNERS]: Done

**Pull request merged**